external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -125,50 +125,52 @@ The `from_existing` value when set provides the basis of linked data; it allows
+| `type`                   | The type of the column; to configure an attribute column use the value `attribute` (Required)                                                                                  | *dimension*                                                                      |
+| `label`                  | The title of the column (Optional)                                                                                                                                             | The capital case of the header in the csv file with spaces replacing underscores |
+| `description`            | A description of the contents of the column (Optional)                                                                                                                         | *none*                                                                           |
+| `from_existing`          | The uri of the resource for reuse/extension (Optional)                                                                                                                         | *none*                                                                           |
+| `definition_uri`         | A uri of a resource to show how the column is created/managed (i.e. a uri of a PDF explaining a list of units) (Optional)                                                      | *none*                                                                           |
+| `data_type`              | The [xml data type](https://www.w3.org/TR/xmlschema-2/#built-in-datatypes) of the contents of the column, if this is provided it becomes a Literal Attribute column (Optional) | *none*                                                                           |
+| `required`               | If this boolean value is true csvcubed will flag to the user if there are blank values in this column                                                                          | *none*                                                                           |
+| `from_template`          | Use a [column template](templates.md)                                                                                                                                          | *none*                                                                           |
+| `describes_observations` | Associates observation values with the relevant attribute. This is only necessary for [pivoted shape data sets](../shape-data/pivoted-shape.md) with multiple observation value columns.                                                                                                                      | *none*                                                                           |
```

Comment:
I would be more tempted to say `Associated this attribute with the relevant observation values` - and the same with the units one too.

Reply:
Done